src/app/pages/onboarding/set-password/components/password-bars.tsx

```diff
@@ -0,0 +1,24 @@
+import { styled } from 'leather-styles/jsx';
+
+interface PasswordStrengthBarsProps {
+  bars: string[];
+}
+export function PasswordStrengthBars({ bars }: PasswordStrengthBarsProps) {
+  return (
+    <styled.div display="flex" flexDirection="row" height="6px">
```

Comment:
Would use `<Flex />` and `<Box />` here imo as the div and span don't actually present any worthy semantic meaning

Reply:
I tried to change this but noticed errors: https://github.com/leather-wallet/extension/pull/4519#issuecomment-1812532846 

I'll merge this and try change when fixing that